okhttp/src/commonMain/kotlin/okhttp3/internal/-CommonPlatform.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okhttp3.internal
+
+expect enum class TimeUnit
```

Comment:
I don’t think it’s reasonable for us to put public APIs in our `internal` package

Reply:
That was a mistake, but no longer relevant.  Agree with you on this being public API.  We aren't the owners of TimeUnit.  I think okio.IOException is justifiable.  But we aren't OkTime.